### ISSUE TITLE
No module named 'packaging'

### ISSUE THREAD

**Issue by efojs** [pyepye/django-magiclink#6]
Just installed and run `runserver`  

Get:  
```bash
$ python manage.py runserver
Watching for file changes with StatReloader
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/Users/USERNAME/.pyenv/versions/3.8.6/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
...
File "/Users/USERNAME/.local/share/virtualenvs/project-G1DqfJzJ/lib/python3.8/site-packages/magiclink/__init__.py", line 2, in <module>
    from packaging import version
ModuleNotFoundError: No module named 'packaging'
```

**What am I doing wrong?**
Django: 3.2.4

**Comment by efojs** [comment]
Found similar issue: https://github.com/PayneLab/cptac/issues/3  
Installing `packaging` manually fixed it  

But do I need it though? Seems like it's required for [writing reusable apps](https://docs.djangoproject.com/en/3.2/intro/reusable-apps/#packaging-your-app)  
Looks like it should not be a dependency, should it?

**Comment by pyepye** [comment]
Hi @efojs 

Yep looks like this is a break in the current build. `packaging` used to be automatically installed via Django but no longer is a dependency. 
I'm fixing now to not require it. Sorry about that.

**Comment by pyepye** [comment]
Hi @efojs 

Version [`1.0.3` has been released](https://github.com/pyepye/django-magiclink/releases/tag/1.0.3). It's also been released to [PyPi](https://pypi.org/project/django-magiclink/) but can take a while to propagate so pip install might install 1.0.2 for another few minutes. 

I decided it keep the `packaging` dependency so ensure dlevel versions etc were handled properly, I've just made sure it's actually added as a dependency in `pyproject.toml`.

I'm still not sure why `tox` didn't catch this mind you.

Again sorry the issue.

Matt
